typings/bn.d.ts

```diff
@@ -0,0 +1,510 @@
+// Type definitions for bn.js 4.11
```

Comment:
you could use types from https://www.npmjs.com/package/@types/bn.js with simple setup in tsconfig.js and package.json

Reply:
Yes, this is done because there is a small error in that version of the file :)